src/test/java/com/xceptance/neodymium/testclasses/data/RandomnessOfDataSets.java

```diff
@@ -30,6 +33,7 @@ public void test1()
     @SuppressDataSets
     public void test2()
```

Comment:
rename to testOrderOfChosenDataSetsHasChanged

Reply:
Done